### PR TITLE
Update docs_style_guide.md

### PR DESCRIPTION
**Pull request by manez**
**GitHub Issue**: #1040 
* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?

Fixes the tip about how to make Tips so that it should actually render properly in the docs Right now, [it's not](https://islandora.github.io/documentation/contributing/docs_style_guide/).

# What's new?
I think it needs a title to work properly.

# How should this be tested?

run mkdocs on the docs and see if the tip presents as a nice blue box, as seen [here](https://islandora.github.io/documentation/user-documentation/media/)

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

**Review by rosiel — APPROVED** [review]

File: docs/contributing/docs_style_guide.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
    -- For file name, use underscores between words and prefix all file names with the page name, e.g. context_display_hints.jpg for the image showing how to set display hints in the context menu.
 - Use `!!! note` at the start of a paragraph to have it rendered as a note like this:
```
Comment (whikloj) [inline docs/contributing/docs_style_guide.md]: Maybe this should specify that you need to have a title, as it appears required. Also might be good to explictly display the note below.
> Use `!!! "Note Title"` at the start of a paragraph to have it rendered as a note like this:
> ```
> !!! "Helpful Tip" I am a helpful tip!
> ```

Comment (manez) [reply docs/contributing/docs_style_guide.md]: Good idea. Done!

**Review by whikloj — APPROVED** [review]
👍

**Comment by whikloj** [comment]
@manez your branch is out-of-date, could you update so we can merge. It is my arch-nemesis.

**Comment by manez** [comment]
@whikloj all up to date, I think?

**Pull request merged**